migrate/live-migration/live-migration-from-postgres.md

```diff
@@ -12,402 +12,158 @@ import StepOne from "versionContent/_partials/_migrate_dual_write_step1.mdx";
+evident when the lag outputs 0 megabytes.
 
-## 8. Promote target database as new primary
-
-In this step you will switch production traffic over to the new database. In
-order to achieve transactional consistency, your application must stop writing
-data to the source database, and all writes should be flushed from the source
-to the target before your application starts writing to the target database.
-
-This means that your application will experience partial downtime (in writes).
-The duration of this downtime is proportional to the steady-state `replay_lag`.
-Ideally, this is on the order of seconds or minutes.
-
-Before you proceed, ensure that you have checked the current value of
-`replay_lag`, and that you are comfortable with a partial downtime of this
-duration.
-
-Update the table statistics by running `ANALYZE` on all data:
+When the lag between the two databases shows 0 megabytes, wait for a
+few minutes and then provide the signal to stop the live replay by
+pressing key `s` or giving a `SIGUSR1` signal, as per your log.
+You should see `Stopped` message.
 
 ```sh
-psql $TARGET -c "ANALYZE;"
+[WATCH] Source DB - Target DB => 0MB. Press 's' (and ENTER) to stop live-replay
+Stopped
 ```
 
-<Highlight type="note">
-Application downtime begins here
-</Highlight>
-
-Stop the write traffic to the source database. With no new data being ingested,
-the `replay_lag` should decrease considerably faster.
+Now, compare tables in the target against those in the source to ensure
+that both have the same data.
 
-Instruct `pgcopydb` to stop when it's done applying the changes that have been
-generated up to this point:
+Once you are confident that the tables in the source are identical to the
+target, you need to signal “continue”. This is done either by pressing
+key `c` or giving a `SIGUSR1` signal, as mentioned in your log.
 
 ```sh
-pgcopydb stream sentinel set endpos --current --source $SOURCE
+[ACTION NEEDED] Now, you should check the integrity of your data. Once you are confident, press 'c' (and ENTER) to continue
+Continuing ...
+Syncing last LSN in Source DB to Target DB ...
 ```
 
-Data written to the source database after this point won't be replicated.
-
-Changes to sequences are not replicated. Fortunately, `pgcopydb` has a command
-to migrate them:
+The live migration image will continue the remaining work that includes
+migrating sequences and running policies. You should see the following
+message if all the mentioned steps were successful.
 
 ```sh
-pgcopydb copy sequences \
-  --source "$SOURCE" \
-  --target "$TARGET" \
-  --resume \
-  --not-consistent
+Migration successfully completed
 ```
 
-Wait until the `pgcopydb follow` process from [step 3][step-3] finishes its
-execution. The process runs until the end position is reached. If you started
-`pgcopydb follow` in the background you can bring it to the foreground with the
-`fg` command.
+## 4. Validate the data in target database and promote as new primary
```

Comment:
I don't really like the word "promote" here, it implies doing something to the database, which we're not doing.
```suggestion
## 4. Validate the data in target database and use it as new primary
```